sys/include/net/netdev2_test.h

```diff
@@ -0,0 +1,306 @@
+/*
```

Comment:
As this is quite gnrc related, I think it would make more sense to locate this file somewhere under **sys/include/net/gnrc/* *